### ISSUE TITLE
two questions about paper

### ISSUE THREAD

**Issue by sigma-alpha-beta** [fabvio/TuSimple-lane-classes#3]
Hi， thanks for your work, There are two things I don't quite understand about the paper
1、“we train the CNN to recognize lane boundaries,  rather  than  lane  markings. ”，Does Lane boundaries mean all the lanes（incuding dashed and continuous）? what is the lane  markings mean?
2、“we perform instance segmentation on lane boundaries instead of semantic segmentation. In this way, it is possible to distinguish different lane boundaries without relying on clustering algorithms. ”， But it's mentioned later “We choose ERFNet [19] as our baseline mode”， Isn't erfnet a semantic segmentation model?
thanks!

**Comment by fabvio** [comment]
Hi, thanks for your interest.
1. We define as "lane markings" every paint markings on the road. "Lane boundaries", instead, are the actual borders of a lane. Typically, a single boundary can encompass both markings and simple asphalt, as it happens with dashed lanes.
2. Erfnet is indeed a semantic segmentation model, but we only exploited its architecture to perform instance segmentation. Basically, what we do is first pretraining it on the binary segmentation problem (lane boundary / background) then finetune it using a clustering loss (implementation [here](https://github.com/fabvio/Cascade-LD/blob/master/loss.py)). 

Feel free to reopen the issue if you have additional questions :)
